File: src/main/java/com/github/sheauoian/croissantsys/common/user/UserRunnable.java
package com.github.sheauoian.croissantsys.common.user;

import com.github.sheauoian.croissantsys.CroissantSys;
import org.bukkit.scheduler.BukkitRunnable;

public class UserRunnable extends BukkitRunnable {
    @Override
    public void run() {
        for (OnlineUser onlineUser: UserManager.getInstance().getOnlineUsers()) onlineUser.update();
    }
}

File: src/main/java/com/github/sheauoian/croissantsys/user/UserRunnable.kt
package com.github.sheauoian.croissantsys.user

import com.github.sheauoian.croissantsys.CroissantSys
import org.bukkit.scheduler.BukkitRunnable

class UserRunnable: BukkitRunnable() {
    override fun run() {
        CroissantSys.instance.server.onlinePlayers.forEach {
            UserDataManager.instance.getOnline(it)?.update()
        }
    }
}